src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -565,9 +569,9 @@ uct_ib_mlx5_devx_reg_mt(uct_ib_md_t *ibmd, void *address, size_t length,
         goto err_free;
     }
 
-    status = uct_ib_mlx5_devx_reg_ksm_data(md, is_atomic, ksm_data, length, 0,
-                                           "multi-thread key", &ksm_data->dvmr,
-                                           &mkey);
+    status = uct_ib_mlx5_devx_reg_ksm_data(md, is_atomic, address, ksm_data,
```

Comment:
seems like number of parameters can be reduced for this func and for `uct_ib_mlx5_devx_reg_ksm_data_contig` if pass memh

Reply:
it's the prev version, would need to pass mr_type